MIGRATING.md

```diff
@@ -99,6 +99,8 @@ A response resolver now exposes a single object argument instead of `(req, res,
 To mock responses, you should now return a Fetch API `Response` instance from the response resolver. You no longer need to compose a response via `res()`, and all the context utilities have also [been removed](#context-utilities).
 
 ```js
+import { http, Response } from 'msw'
```

Comment:
`Response` is a global class, we don't need to import it. Besides, `msw` doesn't export it, to begin with :D You can drop it.

Reply:
Ah, fair! In that case there are some instances from before my PR which I'll drop too. :smile: